pkg/kubelet/container/cache.go

```diff
@@ -104,6 +104,7 @@ func (c *cache) Set(id types.UID, status *PodStatus, err error, timestamp time.T
 	defer c.lock.Unlock()
 
 	if utilfeature.DefaultFeatureGate.Enabled(features.EventedPLEG) {
+		// Added after Evented PLEG was added
```

Comment:
This is the only place except `kubelet.go` to calling the FG status. See https://github.com/kubernetes/kubernetes/pull/122778#discussion_r1459966444 for why we keep it.

- Using `pleg.InUse()` will make a import cycle.

Reply:
/hold 

In light of this PR, https://github.com/kubernetes/kubernetes/pull/120942/files, I am wondering if we need the code within this if block at all. 

I will run some jobs removing this code and get back here. 

https://github.com/kubernetes/kubernetes/pull/122737/files#diff-b010d1c9ff51c8975875be97aeebd3c8c6206128f520115752f9dc007e8afcd1R104

Reply:
@pacoxu we can remove this block altogether,

https://github.com/kubernetes/kubernetes/pull/122737/files#diff-b010d1c9ff51c8975875be97aeebd3c8c6206128f520115752f9dc007e8afcd1R104

`pull-kubernetes-node-crio-cgrpv1-evented-pleg-e2e` [passes](https://github.com/kubernetes/kubernetes/pull/122737#issuecomment-1904283564) fine without that block.

Reply:
Removed.

Reply:
Thanks. 

/hold cancel